cloud/linode/cilium_loadbalancers.go

```diff
@@ -87,6 +88,22 @@ func (l *loadbalancers) getExistingSharedIPs(ctx context.Context) ([]string, err
 	return addrs, nil
 }
 
+func (l *loadbalancers) getExistingSharedIPsOnHolder(ctx context.Context) ([]string, error) {
+	addrs := []string{}
+	ipHolder, err := l.ensureIPHolder(ctx)
+	if err != nil {
+		return addrs, err
+	}
+	ipHolderAddrs, err := l.client.GetInstanceIPAddresses(ctx, ipHolder.ID)
+	if err != nil {
+		return addrs, err
+	}
+	for _, addr := range ipHolderAddrs.IPv4.Shared {
+		addrs = append(addrs, addr.Address)
+	}
```

Comment:
```suggestion
	addrs := make([]string, 0, len(ipHolderAddrs.IPv4.Shared))
	copy(addrs, ipHolderAddrs.IPv4.Shared)
```

Reply:
`ipHolderAddrs.IPv4.Shared` is type `[]*InstanceIP` so I have to keep the for loop to construct the slice of strings

Reply:
D'oh, sorry. I totally missed that. :face_exhaling: